content/using-atom/sections/basic-customization.md

```diff
@@ -225,6 +225,12 @@ The scope name is shown in the settings view for each language. Click on "Packag
 
 ![Finding a language grammar](../../images/python-grammar.png "Finding a grammar's scope name")
 
+Another way to find the scope for a specific language is to open a file of it's kind and press <kbd class="platform-mac">Alt+Cmd+P</kbd><kbd class="platform-windows">Alt+Ctrl+Shift+P</kbd><kbd class="platform-linux">Ctrl+Alt+Shift+P</kbd> to show all scopes for the current position of the cursor.
```

Comment:
`open a file of its kind`
`<kbd class="platform-windows platform-linux">Alt+Ctrl+Shift+P`